app/src/main/res/layout/recycler_sensor_list_item.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
just to let you know, there is method to make all the recycler view to use data binding. If you would want to make it in the future